example/android/app/build.gradle

```diff
@@ -38,12 +38,11 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "ar.com.p39.mercado_pago_mobile_checkout_example"
-        minSdkVersion 21
-        targetSdkVersion 28
+        applicationId "com.example.example"
+        minSdkVersion 16
```

Comment:
I think 16 is too low for px-android, lets keep it in 21.

Reply:
Yeah, my bad. I didn't test this PR for Android as no changes should affect it. That made me ignore the `example`'s Android part.